cl/opinion_page/templates/opinion.html

```diff
@@ -201,11 +201,22 @@ <h3><span>Related Opinions</span></h3>
         {% if cluster.top_visualizations %}
             <div id="visualizations" class="sidebar-section">
                 <h3><span>Visualizations</span></h3>
-                {% url "cluster_visualizations" pk=cluster.pk slug=cluster.slug as cluster_visualizations_url %}
 
-                {% with opinions=cluster.top_visualizations|slice:":5" full_list_url=cluster_visualizations_url|add:"?q"|add:request.META.QUERY_STRING %}
-                    {% include 'includes/opinions_sidebar.html' %}
-                {% endwith %}
+                <ul>
+                    {% for viz in cluster.top_visualizations|slice:":5" %}
+                        <li>
+                            <a href="{{ viz.get_absolute_url }}">
+                                {{ viz.title|default_if_none:"N/A"|safe|truncatewords:10|v_wrapper }}
+                            </a>
+                        </li>
+                    {% endfor %}
+                </ul>
+                <p>
+                    <a href="{% url "cluster_visualizations" pk=cluster.pk slug=cluster.slug %}"
+                       class="btn btn-default">
+                        View All Visualizations
```

Comment:
Also omitted from this and other links is `?{{ request.META.QUERY_STRING }}`, which is widely used for other sidebar links.

For the “View All” two q’s had been added (`?qq=`) this has got to be a mistake. For the visualization links themselves, there doesn’t appear to be any purpose served by the query.

This said, I haven’t been able to explain to myself the reason for `?{{ request.META.QUERY_STRING }}` on the opinion links. Is it a vestigial thing? part of (now lost) breadcrumb feature? Could it be stripped from all links in a follow-up?

Reply:
Yeah, no point adding the GET parameters to viz links. 

As for the opinion links, I keep those params around in the hopes that one day we'll use them for highighting the search terms in the text of the opinions, but that day may never come. This could certainly be stripped.